src/HttpRequest.cc

```diff
@@ -615,8 +615,8 @@ HttpRequest::getRangeOffsetLimit()
     for (AclSizeLimit *l = Config.rangeOffsetLimit; l; l = l -> next) {
         /* if there is no ACL list or if the ACLs listed match use this limit value */
         if (!l->aclList || ch.fastCheck(l->aclList).allowed()) {
-            debugs(58, 4, HERE << "rangeOffsetLimit=" << rangeOffsetLimit);
             rangeOffsetLimit = l->size; // may be -1
+            debugs(58, 4, HERE << "rangeOffsetLimit=" << rangeOffsetLimit);
```

Comment:
Please remove `HERE <<` from the arguably modified debugs() line. We are trying to get rid of that macro when we have to change debugging for some other reason.